Client.xcodeproj/project.pbxproj

```diff
@@ -4872,6 +4886,9 @@
 				"$(SRCROOT)/Carthage/Build/iOS/SwiftyJSON.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/KSCrash.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/Sentry.framework",
+				"$(SRCROOT)/Carthage/Build/iOS/Leanplum.framework",
+				"$(SRCROOT)/Carthage/Build/iOS/LeanplumLocationAndBeacons.framework",
```

Comment:
Yes we can. But weirdly even though the Cartfile only pulls in the LP framework, Carthage still downloads the two other frameworks.